config/routes.rb

```diff
@@ -3,6 +3,5 @@
   root 'dashboards#index'
 
   devise_for :users
-  resources :foods
-
+  resources :foods, only: %i[index show new create destroy]
```

Comment:
we don't need to specify all actions as long as the resources: foods as the s at the end.
it should be  `resources :foods`

Reply:
Hey @vic778 thanks for the review , but can't see that in my local machine , you can now check all is ok